docs/incubating/custom-evaluator.md

```diff
@@ -0,0 +1,70 @@
+# Custom violations evaluator (incubating)
+
+> Since: [`0.5.1`](https://github.com/novoda/gradle-static-analysis-plugin/releases/tag/v0.5.1)
+>
+> :warning: This is an **experimental**, incubating feature that may be subject to breaking API changes at any time!
+
+The plugin uses a [`ViolationsEvaluator`][violationsevaluatorcode] to determine what to do with the results collected from all the active
+tools (if any). The built-in behaviour is provided by the [`DefaultViolationsEvaluator`][defaultviolationsevaluatorcode], which you can
+read more about [below](#the-defaultviolationsevaluator). The plugin's violations evaluation behaviour is not fixed, and it can be
+customised by providing an implementation of the [`ViolationsEvaluator`][violationsevaluatorcode] interface.
+
+## Table of contents
+ * [The `DefaultViolationsEvaluator`](#the-defaultviolationsevaluator)
+ * [Creating a custom violations evaluator](#creating-a-custom-violations-evaluator)
+
+---
+
+## The `DefaultViolationsEvaluator`
+The plugin has a default mechanism to decide whether to consider a build as passing or failed. The mechanism is manifesting itself
+as the `penalties` closure:
```

Comment:
it's actually `penalty`

Reply:
I _knew_ I'd misspell it LOL

Reply:
cf03f2b9e4c71d7df49f7a07111715bb0ee96957